README.md

```diff
@@ -1,157 +1,159 @@
+   For information about instrumentation names, see
+   [Supported libraries](#supported-libraries). If you configure Rails
+   instrumentation, it also configures Active Record instrumentation, so you don't
+   need to instrument both.
+
+### Manually install the library
+
+1. Download the [latest release](https://github.com/signalfx/signalfx-ruby-tracing/releases/latest) of the tracing library.
+2. Add `signalfx-tracing` to your application's Gemfile:
+   ```bash
+   $ gem 'signalfx-tracing'
+   ```
+3. Add each applicable instrumentation to your application's Gemfile. For
+   example, this is how you add Rails and Redis:
+   ```bash
+   $ gem 'rails-instrumentation'
+   $ gem 'redis-instrumentation'
+   ```
+   For information about instrumentation names, see
+   [Supported libraries](#supported-libraries). If you configure Rails
+   instrumentation, it also configures Active Record instrumentation, so you don't
+   need to instrument both.
+4. Install the gems for the tracing library and instrumentations:
+   ```bash
+   $ bundle install
+   ```
+
+## Configure instrumentation for a Ruby application
+
+Configure the instrumentation anywhere in the setup portion of your code or
+before doing anything that needs to be traced.
+
+For example, with Rails, configure instrumentation in `config/initializer/tracing.rb`.
+
+You can configure instrumentation automatically or manually. Manual
+instrumentation is convenient when you want to trace only some libraries.
+
+### Set configuration values
+
+If the default configuration values don't apply for your environment, override them before running the process you instrument.
+
+| Configure parameter | Environment variable  | Default                          | Notes |
+| ------------------- | --------------------  | -------------------------------- | ----- |
+| tracer              | N/A                   | `nil`                            | The OpenTracing global tracer. |
+| ingest_url          | SIGNALFX_ENDPOINT_URL | `http://localhost:9080/v1/trace` | The endpoint the tracer sends spans to. Send spans to a Smart Agent, OpenTelemetry Collector, or a SignalFx ingest endpoint. |
+| service_name        | SIGNALFX_SERVICE_NAME | `signalfx-ruby-tracing`          | The name to identify the service in SignalFx. |
+| access_token        | SIGNALFX_ACCESS_TOKEN | `''`                             | The SignalFx organization access token. |
+
+### Automatically instrument code:
+
+Configure the auto-instrumentor to check for modules defined in the code and
+instrument them if available:
 
 ```ruby
-gem 'signalfx-tracing'
-```
-
-Then execute the following (or use your desired installation method for your application).
+require 'signalfx/tracing'
 
-```bash
-$ bundle install
+SignalFx::Tracing::Instrumenter.configure(auto_instrument:true)
 ```
 
-## Usage
-
-Configure the instrumentation anywhere in the setup portion of your code or before doing anything
-that needs to be traced.
+### Manually instrument code:
```

Comment:
Don't think this requires a separate section since they all involve auto-instrumentation.  One uses the all library helper and the other explicitly enumerates the desired target libraries.

Reply:
is there a better way to differentiate between the two methods?

Reply:
I think removing the `Manually instrument code:` header and updating "Specify which libraries to instrument" to something like "Or you can specify which target libraries to instrument:"